reputation/rpc/api.proto

```diff
@@ -1,10 +1,7 @@
 syntax = "proto3";
-package filecoin.reputation.pb;
+package reputation.rpc;
 
-option java_multiple_files = true;
-option java_package = "io.textile.filecoin.reputation.pb";
-option java_outer_classname = "FilecoinReputation";
-option objc_class_prefix = "TTE";
+option go_package = "rpc";
```

Comment:
Useful option I didn't know about. Useful because you really do need to make the `package` above unique and consistent with the directory the .proto file is in. This isn't a problem in Go, but is for JS and other languages.